metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/utils/tool.clj

```diff
@@ -0,0 +1,33 @@
+(ns cmr.metadata-db.int-test.concepts.utils.tool
+  "Defines implementations for all of the multi-methods for tools in the metadata-db
+  integration tests."
+  (:require
+   [cheshire.core :as json]
+   [cmr.metadata-db.int-test.concepts.utils.interface :as concepts]))
+
+(def tool-json
```

Comment:
nit: this should only define a map and `json/generate-string` should be in the `get-sample-metadata` function on line 15.